deploy/bundle/generate-csv-template.py

```diff
@@ -47,6 +54,15 @@
 csv['spec']['install']['spec']['deployments'][0]['spec']['template']['spec']['containers'][0]['image'] = \
     '${IMAGE}:${IMAGE_TAG}'
 
+if args.replaces:
+    csv['spec']['replaces'] = f'deployment-validation-operator.v{args.replaces}'
+
+if args.skip:
+    csv['spec']['skips'] = [
```

Comment:
Does `skipRange` work with `replaces`? It was my understanding it didn't or at least it looks to me as a weird mixture. Since we want to continue using `replaces` I prefer to use `skip`

Reply:
iirc, semver mode ignores `replaces` and relies on the ordering based upon the operator's version field.  It's a bit deceptive.  Semver mode uses the operator.version field to determine updates.  Non-semver mode use operator.name and replaces (which keys off operator.name) to determine upgrade chain.  They are similar, but different in important ways.

Reply:
Right; so using `replaces` *and* `skipRange` would work for both variants. Non-semver will observe `replaces` and `skips`/`skipRange`; but semver will ignore `replaces` and `skips` but observe `skipRange`. So the most generic and future-proof variant would be to use `replaces`+`skipRange`.